spacy/kb.pyx

```diff
@@ -0,0 +1,147 @@
+# cython: profile=True
+# coding: utf8
+from spacy.errors import user_warning
+
+
+cdef class Candidate:
+
+    def __init__(self, KnowledgeBase kb, entity_id_hash, alias_hash, prior_prob):
+        self.kb = kb
+        self.entity_id_hash = entity_id_hash
+        self.alias_hash = alias_hash
+        self.prior_prob = prior_prob
+
+    property entity_id:
```

Comment:
Super nitpicky, but now that Cython supports the `@property` syntax, we recently came up with our own convention for this: If there's only a getter, we use `@property` with `def`, if there's a getter and setter, we use `property foo:`.

Reply:
Please be as nitpicky as you like, especially since it's my first big contribution in Cython, and I don't know all the conventions yet!

Reply:
We don't have so many code style conventions for Cython yet (except for [these](https://spacy.io/api/cython#conventions), which you've probably seen already) – also because auto-formatting and linting is still a bit difficult for Cython.

(For Python, `black` and `flake8` enforce most of the standards, so that's much easier. With the current version, you should be able to just run `flake8 spacy` and it shouldn't produce any warnings. And if you do have to break one of the rules on purpose, like, assign an unused variable, you can add `# noqa: F841` etc to the line.)